src/qt/clientmodel.cpp

```diff
@@ -15,6 +15,7 @@
 #include <net.h>
 #include <netbase.h>
 #include <util/system.h>
+#include <validation.h>
```

Comment:
I don't think you're supposed to introduce a direct dependency here, either the flag should be passed in or the request should go through the node interface
@ryanofsky

Reply:
ACK, so use the node `getFreindex()` method.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/17565#discussion_r349868075

You'd replace `fReindex` here with `clientmodel->node().getReindex()`